src/python/WMCore/Services/DBS/DBSErrors.py

```diff
@@ -96,6 +96,17 @@ def getServerCode(self):
             return self.data['error']['code']
         return 0
 
+    def getCodes(self):
+        """
```

Comment:
https://github.com/dmwm/WMCore/blob/master/CONTRIBUTING.rst#project-docstrings-best-practices

Reply:
I do not get the comment here.... @amaltaro, what is not followed in this docstring by the book?

Reply:
I would leave the description as is, given that the function is not parsing and then separately putting the results in a list, but it's creating a list making the associations between the error messages and the error codes

Reply:
I think the implementation of the function/method does not influence the structure of the docstring.
But if people want to update the contribution guidelines with a different recommendation, let us do it. 
I am just trying to seek consistency in the project.